src/screens/ContactScreen/index.js

```diff
@@ -262,9 +280,8 @@ const styles = StyleSheet.create({
   containerStyle: {
     marginHorizontal: 16
   },
-  subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
-    marginEnd: 8
+  subtitleStyle: (sourceScreen) => ({
+    color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `subtitleStyle` function is returning an object with a single property. It would be more efficient to return the color directly and use it as needed.
```diff
- subtitleStyle: (sourceScreen) => ({
-   color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
- })
+ subtitleStyle: (sourceScreen) => sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->